src/Tes.ApiClients/RetryHandler.cs

```diff
@@ -132,3 +166,25 @@ public virtual async Task<HttpResponseMessage> ExecuteHttpRequestWithRetryAsync(
         return await asyncHttpRetryPolicy.ExecuteAsync(action, cancellationToken);
     }
 }
+
+public static class RetryHandlerExtensions
```

Comment:
What is the purpose of these methods, I don't see them used anywhere? Perhaps, I missed it.